algo/src/test/java/com/hartwig/serve/sources/ckb/blacklist/CkbBlacklistEvidenceTest.java

```diff
@@ -0,0 +1,355 @@
+package com.hartwig.serve.sources.ckb.blacklist;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import com.hartwig.serve.datamodel.EvidenceLevel;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.Test;
+
+public class CkbBlacklistEvidenceTest {
+
+
+    @NotNull
+    public static CkbEvidenceBlacklistModel defineEvidenceModel(@NotNull CkbBlacklistEvidenceType type, @Nullable String therapy,
+            @Nullable String cancerType, @Nullable String gene, @Nullable String event, @Nullable EvidenceLevel level) {
+        List<CkbBlacklistEvidenceEntry> blacklistStudiesList = Lists.newArrayList();
+        CkbBlacklistEvidenceEntry entry1 = ImmutableCkbBlacklistEvidenceEntry.builder()
+                .type(type)
+                .therapy(therapy)
+                .cancerType(cancerType)
+                .gene(gene)
+                .event(event)
+                .level(level)
+                .build();
+
+        blacklistStudiesList.add(entry1);
+        return new CkbEvidenceBlacklistModel(blacklistStudiesList);
+    }
+
+    @NotNull
+    public static CkbEvidenceBlacklistModel createCkbBlacklistEvidence() {
```

Comment:
There should not be public static factory methods in an individual test class (see other comments)

Reply:
To set this function private isn't possible because another class has this needed

Reply:
Have refactored, check my recent commit!